autoemulate/experimental/data/utils.py

```diff
@@ -165,6 +171,18 @@ def _random_split(
         test_loader = self._convert_to_dataloader(test, batch_size=batch_size)
         return train_loader, test_loader
 
+    @staticmethod
+    def _normalize(x: TensorLike) -> tuple[TensorLike, TensorLike, TensorLike]:
+        x_mean = x.mean(0, keepdim=True)
+        x_std = x.std(0, keepdim=True)
+        return (x - x_mean) / x_std, x_mean, x_std
+
+    @staticmethod
+    def _denormalize(
+        x: TensorLike, x_mean: TensorLike, x_std: TensorLike
+    ) -> TensorLike:
+        return (x * x_std) + x_mean
```

Comment:
Added this here as initial functionality form preprocessing outputs. This would be good to revisit in #348 where this might be better as distinct functionality that can be combined with the model and/or updated with the API used there.

Reply:
Also relates to #437